package.json

```diff
@@ -1,18 +1,25 @@
 {
-  "name": "webrtc-signal-http",
-  "version": "1.7.0",
+  "name": "webrtc-signal-http-ts",
```

Comment:
blocking on reverting this name, and semver-ing correctly for merge to this repo.